src/main/java/org/iot/dsa/dslink/dynamodb/DynamoDBDSAClient.java

```diff
@@ -0,0 +1,146 @@
+package org.iot.dsa.dslink.dynamodb;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.model.ListTablesResult;
+
+import org.iot.dsa.node.DSMap;
+
+import java.util.List;
+
+public class DynamoDBDSAClient {
+
+    private String accessKey = null;
+    private String secretKey = null;
+    private String region = null;
+    private String endpoint = null;
+    private AmazonDynamoDB client;
+
+    public DynamoDBDSAClient(String accessKey, String secretKey, String region, String endpoint){
+        this.accessKey = accessKey;
+        this.secretKey = secretKey;
+        this.region = region;
+        this.endpoint = endpoint;
+        client = Util.connectDynamoDB(accessKey,secretKey,region,endpoint);
+    }
+
+    public void listtable(){
+        ListTablesResult tables = client.listTables();
+        List tnames = tables.getTableNames();
+        for(int i = 0 ; i < tnames.size();i++){
+            System.out.println(tnames.get(i));
```

Comment:
System.out.println shouldn't be used. If necessary to write to the log, extend DSLogger and call one of the logging methods.

Reply:
ok

Reply:
Done!

Reply:
There are still some System.out.println calls in Util.java, could you change those as well?

Reply:
Done!